app/util.js

```diff
@@ -257,6 +257,42 @@ module.exports = {
     return false;
   },
 
+  renderMedia: function(data, elem, controller, back) {
```

Comment:
I know this function is now repurposed to be called from two places, so I'm glad you put it in a util. However, I don't love the idea of passing around references to jQuery objects. Between different files, it gets a lot harder to know what's being referenced. Is there some way we can avoid this?